libcontainer/selinux/selinux.go

```diff
@@ -158,12 +158,14 @@ func Setfilecon(path string, scon string) error {
 // Getfilecon returns the SELinux label for this path or returns an error.
 func Getfilecon(path string) (string, error) {
 	con, err := system.Lgetxattr(path, xattrNameSelinux)
-
+	if err != nil {
+		return "", err
+	}
 	// Trim the NUL byte at the end of the byte buffer, if present.
 	if con[len(con)-1] == '\x00' {
```

Comment:
What are the odds of no error and len(con)==0? If possible should probably add a if len(con) check before causing a "panic: runtime error: index out of range"